### PR TITLE
sidebar_user_setting_logout

### PR DESCRIPTION
**Pull request by AkromSaidxonov**
logout and user settings section of the sidebar section has been prepared

### CODE REVIEW

File: src/components/Sidebar/Logout.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      dangerMode: true,
+    }).then((willLogout:any) => {
+      if (willLogout) {
+        window.localStorage.removeItem("token");
+        navigate("/login");
+      } else {
+      }
+    });
+  };
+  //
```
Comment (AliWayDev) [inline src/components/Sidebar/Logout.tsx]: comment delete

File: src/components/Sidebar/Logout.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import LogoutIcon from "@mui/icons-material/Logout";
+import { useNavigate } from "react-router-dom";
+import swal from "sweetalert";
+
+import Box from "@mui/material/Box";
+
+const Logout = () => {
+  const navigate = useNavigate();
```
Comment (AliWayDev) [inline src/components/Sidebar/Logout.tsx:9]: space kerak boshqa joylarni ham ko'ring

**Comment by AkromSaidxonov** [comment]
image

File: src/components/Sidebar/SidebarFooter.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+                },
+              },
+            }}
+            transformOrigin={{ horizontal: "right", vertical: "top" }}
+            anchorOrigin={{ horizontal: "right", vertical: "bottom" }}
+          >
+            <SettingMenu />
+          </Menu>
+
+          {/* {isOpen ? (
```
Comment (AliWayDev) [inline src/components/Sidebar/SidebarFooter.tsx]: comment delete

File: src/components/Sidebar/Logout.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        display: "flex",
+        justifyContent: "space-between",
+        alignItems: "center",
+        cursor: "pointer",
+      }}
+      className="logOut"
+      onClick={willLogout}
+    >
+      <LogoutIcon sx={{ transform: "rotate(180deg)", marginRight: "8px" }} />{" "}
+      <h4 className="logOut__title">Log-out</h4>
```
Comment (AliWayDev) [inline src/components/Sidebar/Logout.tsx]: delete classnames

**Review by AliWayDev — CHANGES_REQUESTED** [review]

**Comment by AkromSaidxonov** [comment]
image

**Review by AliWayDev — APPROVED** [review]

**Pull request merged**